12_col_widths.qmd

```diff
@@ -0,0 +1,81 @@
+---
+title: "Setting Column Widths"
+format: html
+editor: visual
+---
+
+## Adjusting column Widths output
+
+
+```{r eval=FALSE}
+
+library(gtsummary)
+tbl <- gtsummary::tbl_summary(gtsummary::trial, by = trt, include = c(age, grade), missing = "always", label = list(age = "Age  \n\U00A0\U00A0linebreak in a cell")) |> add_overall()
+gt_table <- tbl$table_body |> 
+  gt::gt(caption = gt::md("This is the caption  \n[Used for cross referencing in Quarto]")) |> 
+  gt::fmt_markdown(columns = label) |> # need this to recognize the line break in the table body
+  # Column headers with line breaks
+  gt::cols_label(
+    stat_0 = gt::md("**Overall**"),
+    stat_1 = gt::md("**Drug A**  \nManual Line Break"), 
+    stat_2 = gt::md("**Drug B**  \nManual Line Break")
+  ) |> 
+  gt::tab_header(gt::md("This is the Title  \nwith a linebreak"), subtitle = "This is the Subtitle") |>
+  gt::cols_width(everything() ~ px(200)) #apply a 200 px width to all columns
+```
+
```

Comment:
I think it would be helpful to print the table here that we're exporting to Word and PDF. Then we know what the goal is. Can you make that change in all the sections?